examples/image_streamer/golden_images.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright (2012-2016) Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+
+import os
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+
+EXAMPLE_CONFIG_FILE = os.path.join(os.path.dirname(__file__), '../config.json')
+
+oneview_client = OneViewClient.from_json_file(EXAMPLE_CONFIG_FILE)
+
+image_streamer_client = oneview_client.create_image_streamer_client()
+
+# You must set a valid osVolumeURI and buildPlanUri
+golden_image_information = {
+    "type": "GoldenImage",
+    "name": "Demo Golden Image Creation",
+    "description": "Test",
+    "imageCapture": "true",
+    "osVolumeURI": "/rest/os-volumes/57f2d803-9c11-4f9a-bc02-71804a0fcc3e",
+    "buildPlanUri": "/rest/build-plans/b199aada-ed44-4785-8ced-cd979386d4db"
+}
+
+golden_image_upload = {
+    "name": "Demo Golden Image",
+    "description": "Test",
+}
+
+local_image_file_name = 'c:/Util/Licenses.zip'
```

Comment:
Please help me understand why the golden image file is named Licenses.zip.
Does this zip file contain a license or a gold image?